androidapp/app/build.gradle

```diff
@@ -45,8 +45,13 @@ dependencies {
     implementation 'com.google.android.material:material:1.1.0'
     implementation 'androidx.work:work-runtime-ktx:2.3.4'
     implementation 'com.google.code.gson:gson:2.8.5'
+    implementation 'com.squareup.moshi:moshi:1.9.2'
+    implementation 'com.squareup.moshi:moshi-kotlin:1.9.2'
```

Comment:
Do we need Moshi and Moshi Kotlin? I would assume we only need Moshi Kotlin.

- [ ] Determine if we only need one of them or both

Reply:
i believe that both are needed, that the `moshi-kotlin` library is essentially kotlin bindings for the base moshi library